appdatabase/migrations/sql/1733400346_add_rpc_providers.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS rpc_providers (
```

Comment:
As discussed and agreed, we should ensure now that default status-provided networks are not stored in the database anymore, because it makes no sense and only adds confusion. 

We should only persist user-provided secrets here, but the status-provided secrets must remain built-in in the binary.